packs/softlayer/actions/delete_key_pair.yaml

```diff
@@ -0,0 +1,13 @@
+description: 'Deletes a keypair by name. If there are mutiple keys with the same name it will only delete the first'
+enabled: true
+entry_point: run.py
+name: delete_keypair
+parameters:
+  action:
+    default: delete_key_pair
+    immutable: true
+    type: string
+  key_pair:
```

Comment:
Looks like this parameter needs to gold a key pair object name, right?

Please clarify that in the parameter description.